### DISCUSSION TITLE
Access for more powerful machine types

### DISCUSSION THREAD

**Discussion by sr229** [community/community#29351] [Codespaces]
Hey guys! I've been a tester in Codespaces for quite some time now, and I would love to get access to more powerful machines, since I'm doing quite some heavy compile work and GPU-related tasks too. I've sent an email but I have no response yet. Do I need to contact somewhere else?

**Comment by samuel-ma** [comment]
Unfortunately, Github's Codespaces utilizes the full power of your Visual Studio on your hardware

GitHub’s own 35GB dev image starts in under 10 seconds. Scale your cloud VMs up to 32 cores and 64GB of RAM. And with low-latency connections across four regions, you won’t even remember it’s not your local machine.

CPUs up to 32 cores
Memory up to 64 GB
Spin up of less than 10 sec

Read the documentation some more to find a possible middle ground [here](https://www.gitpod.io/docs)
